Gopkg.toml

```diff
@@ -56,7 +56,7 @@
 
 [[constraint]]
   name = "github.com/kata-containers/agent"
-  revision = "8e48125fa2a793706e5e660e95bdc9755f1cf474"
+  revision = "48dd1c031530fce9bf16b0f6a7305979cedd8fc9"
```

Comment:
` vendor/github.com/mdlayher/vsock` has changed, did you forget to include it?

Reply:
No that's fine, it changed because this is a subdependency. The runtime does not directly import the vsock package but the agent does. And by revendoring the agent package, we pull the updated package of vsock.